vividus-util/src/test/java/org/vividus/util/UriUtilsTests.java

```diff
@@ -148,18 +148,18 @@ void testRemoveUnexistentFragment()
         // CHECKSTYLE:OFF
         "http://somehost:8080/path,                                   http://somehost:8080/path",
         "http://somehost:8080/path?name=goodvalue&a=b#fragment,       http://somehost:8080/path?name=goodvalue&a=b#fragment",
-        "http://somehost:8080/pa | th?name=bad|value&a=b#fra| gme nt, http://somehost:8080/pa%20%7C%20th?name=bad%7Cvalue&a=b#fra%7C%20gme%20nt",
+        "http://somehost:8080/pa | th?name=bad|value&a=b#fra| gme nt, http://somehost:8080/pa+%7C+th?name=bad%7Cvalue&a=b#fra%7C%20gme%20nt",
```

Comment:
I'm not sure it's correct to replace space with `+` we need to conform this change with RFC